deployment/helm/eclipse-ditto/Chart.yaml

```diff
@@ -1,4 +1,13 @@
 apiVersion: v1
+appVersion: "1.0.0-M3"
+description: A Helm chart for Eclipse Ditto
 name: eclipse-ditto
-version: 0.9.0
-description: A basic helm chart for Eclipse Ditto and K8s
+version: 0.6.1
```

Comment:
did you see you downgraded the version?

Reply:
Yes. This is the version used in downstream (https://github.com/kiwigrid/helm-charts/tree/master/charts/ditto-digital-twins), to facilitate upcoming changes on the chart.
As the previous chart were nowhere published I think its okay to downgrade the chart version.

Reply:
So you would version the Helm chart differently than the Ditto version?
Sounds reasonable to me, as the chart could e.g. get released more often, even when using the same version of Ditto.

However, using `0.6.1` seems very arbitrary to me ;)
I would suggest starting with `0.7.0`.

Reply:
It is common practice to version the chart independently, therefor the `appVersion` field exists.
Version `0.6.1` is caused by already published versions:

```bash
helm search ditto -l
NAME                        	CHART VERSION	APP VERSION	DESCRIPTION                   
kiwigrid/ditto-digital-twins	0.6.1        	1.0.0-M3   	A Helm chart for Eclipse Ditto
kiwigrid/ditto-digital-twins	0.6.0        	1.0.0-M3   	A Helm chart for Eclipse Ditto
kiwigrid/ditto-digital-twins	0.5.0        	1.0.0-M3   	A Helm chart for Eclipse Ditto
kiwigrid/ditto-digital-twins	0.4.0        	1.0.0-M3   	A Helm chart for Eclipse Ditto
kiwigrid/ditto-digital-twins	0.3.0        	1.0.0-M3   	A Helm chart for Eclipse Ditto
kiwigrid/ditto-digital-twins	0.2.0        	0.9.0      	A Helm chart for Eclipse Ditto
kiwigrid/ditto-digital-twins	0.1.0        	0.9.0      	A Helm chart for Eclipse Ditto
```

As the Chart content provided in this PR are (functional) equal to `kiwigrid/ditto-digital-twins v0.6.1` I prefer to keep it the same.

When we're going to update the chart once Ditto v1.0.0 is released we have to bump the chart version (e.g. to v.0.7.0) in the kiwigrid/helm-chart repo. Afterwards the chart changes will be applied to eclipse/ditto - but how to handle the chart version? I think, this might confuse the users to have equal charts available in different versions.

Reply:
I would propose to distribute the chart via the "Eclipes IoT Packages" project, I created an issue for discussion: https://github.com/eclipse/packages/issues/2

Once we have that in place in the Packages project, I would suggest that you don't longer update your kiwigrid Ditto chart and we will make all improvements to the chart in the "Packages" project.

Reply:
I agree on this.